### DISCUSSION TITLE
Master vs main branch on this repository

### DISCUSSION THREAD

**Discussion by mitchallain** [nvim-treesitter/nvim-treesitter#4876] [General]
This repository contains a `master` and a `main` branch. `main` appears to be 6 commits ahead of `master`. Can we please remove one?

**Comment by amaanq** [comment]
Read the pinned issue.

**Comment by clason** [comment]
And the README on the main branch.

TL;DR: No. Neither will get removed.

**Comment by mitchallain** [comment]
For anyone else who ends up here, this is the pinned issue being referenced:

https://github.com/nvim-treesitter/nvim-treesitter/issues/4767#issue-1698676665 - see the "Plan" subsection.

FWIW, I don't find either of the READMEs particularly instructive as to the (non-standard) branching strategy being used. Would suggest some edits for new users who just want the latest stable version.

**Reply by clason** [reply]
If you want the latest stable version, just use the default branch and don't worry about `main`.
